src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsUserErrors.cs

```diff
@@ -150,38 +145,6 @@ private void Test_Bad_EventSource_Startup(bool onStartup, Listener listener, Eve
                 Assert.Equal(0, ea.Version);
             }
         }
-
-        [Fact]
-        public void Test_Bad_WriteRelatedID_ParameterName()
```

Comment:
This is the only thing that gives me pause.  I wonder if this is already fixed, or if it should be fixed.  I don't recognize the behavior being tested, so I'm wondering if this no longer applies.  @davmason?

Reply:
I just ran it against 8.0, the message is different but we get the error the test expects so this is fixed in 8.0.

Reply:
Awesome.  Glad to hear that.  Should we fix the test?

Reply:
Sure, seems like it would be easer to do as part of this PR - @stephentoub can you re-enable the test? The message I get is `"EventSourceException while processing event \"RelatedActivity\": System.ArgumentException:EventSource expects the first parameter of the Event method to be of type Guid and to be named \"relatedActivityId\" when calling WriteEventWithRelatedActivityId."`

Reply:
I re-enabled it. The expectation is it should "just work"?

Reply:
Thanks, it should just work. If it doesn't just disable it again and I'll deal with it in a separate PR

Reply:
That test passes, but it reliably causes this test to fail:
```
      BasicEventSourceTests.ActivityTracking.StartStopCreatesActivity [FAIL]
        Assert.Equal() Failure: Values differ
        Expected: 00000000-0000-0000-0000-000000000000
        Actual:   c1fc26a8-6f6b-4a0c-9f41-8b29cd5184a3
        Stack Trace:
          D:\repos\runtime\src\libraries\System.Diagnostics.Tracing\tests\BasicEventSourceTest\ActivityTracking.cs(22,0): at BasicEventSourceTests.ActivityTracking.StartStopCreatesActivity()
             at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
          D:\repos\runtime\src\coreclr\System.Private.CoreLib\src\System\Reflection\MethodBaseInvoker.CoreCLR.cs(36,0): at System.Reflection.MethodBaseInvoker.InterpretedInvoke_Method(Object obj, IntPtr* args)
          D:\repos\runtime\src\libraries\System.Private.CoreLib\src\System\Reflection\MethodBaseInvoker.cs(57,0): at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)
```